### ISSUE TITLE
Administrative Data from SQLite

### ISSUE THREAD

**Issue by lhorus** [whosonfirst-data/whosonfirst-data#1717]
I'm attepting to retrieve the administrative data on a global scale, that is, a version containing mostly countries, and one for each detailed country. One option would be to extract each individual country from each database, but considered there are 240-ish countries, this is a bit too much - unless there is no alternative.

However, doing so from the full SQLite database, which is over 28GBs when extracted is a strain on any computer - and mine wont handle it. Is there an easier way to obtain administrative areas ? (Geom, name, relationships)

**Comment by stepps00** [comment]
Thanks for the issue @lhorus 

Who's On First does publish per-country distributions [here](https://dist.whosonfirst.org/sqlite/). Australia, for example, is in the large 28gb "latest" SQLite file at the top of the page, but we also have a "whosonfirst-data-admin-au-latest.db.bz2" distribution that should contain _just_ the administrative data for Australia.

We're still working toward publishing new "bundles" of each placetype [here](https://dist.whosonfirst.org/bundles/), which would allow you to download global coverage of a specific placetype (like all county records, for example). This is still a work in progress, but we hope to get these back in working order shortly.

Let me know of those solutions work or if you have any other questions.

**Comment by lhorus** [comment]
Dear @stepps00 ,

Thank you for your rely. I see there is an old administrative bundle from 2018, I don't mind using it. It should contain all the administrative data for the entire world, correct?

On a side node, any ETA for this bundle placetype?

**Comment by stepps00** [comment]
Sure.. if you're looking for _all_ administrative placetypes in the world, I would suggest using that "whosonfirst-data-latest.db.bz2" file or grabbing each of the per-country SQLIte files for countries of interest here: https://dist.whosonfirst.org/sqlite/

Though, as you said, the global file is fairly large and you may hit issues on your local machine.

If you want to use the bundles ([here](https://dist.whosonfirst.org/bundles/)), they will be out of date and missing much of the new changes to Who's On First since late-2018. Unfortunately there is not estimated time to get the bundles back up and running, but I'm happy to leave this issue open and update it once we do make progress.

**Comment by lhorus** [comment]
Dear @stepps00 ,

I did manage to retrieve the data by mimicking some of the structure in SQL, quite a nice project for leaning WOF.

However, I did notice that administrative levels from 3 onwards are usually stacked as a locality placetype, which invalidates retrieving data for such structure. I noticed this data was being fed from Statoids, but it seems that after its owner's sad passing, the data become private. Is there any recommended source for this information?

**Comment by nvkelso** [comment]
> However, I did notice that administrative levels from 3 onwards are usually stacked as a locality placetype, which invalidates retrieving data for such structure. 

Can you provide examples where there's too many admin levels smushed into the locality placetype in a country, please? Example WOF record IDs would be most helpful.

> I noticed this data was being fed from Statoids, but it seems that after its owner's sad passing, the data become private. Is there any recommended source for this information?

The Statoids data circa 2017 was licensed into Who's On First under CC-BY terms, but sadly hasn't been maintained much since then.

- https://github.com/whosonfirst/whosonfirst-sources/tree/master/sources#statoids

**Comment by lhorus** [comment]
Absolutely, consider the administrative levels list in [Wikipedia](https://en.wikipedia.org/wiki/List_of_administrative_divisions_by_country) for Peru, the third level corresponds to [districts](https://en.wikipedia.org/wiki/Districts_of_Peru). Taking Juliaca as an example, [Spelunker returns only localities](https://spelunker.whosonfirst.org/search/?q=Juliaca).

**Comment by nvkelso** [comment]
(This general support issue is turning into an investigation about Peru... which should probably be split into it's own issue...)

**Comment by Guilherme-B** [comment]
I can give a hand with this subject.

Here's the data for Peru , [link](https://data.humdata.org/dataset/limites-de-peru).

In fact, the same website contains hundreds of resources for multiple countries, [link 2](https://data.humdata.org/search?q=administrative&ext_page_size=100&sort=score%20desc%2C%20max(last_modified%2Creview_date)%20desc).

I can try to give you a hand in mapping these out, there's a lot of countries on that list.

**Comment by stepps00** [comment]
Let's track Peru updates here: https://github.com/whosonfirst-data/whosonfirst-data/issues/1726
